examples/example_dotnet/WinSparkleDemoForm.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+using System.Reflection;
+using System.Threading;
+using System.Windows.Forms;
+using WinSparkleDotNet;
+
+namespace example_dotnet
+{
+    public partial class WinSparkleDemoForm : Form
+    {
+        private SynchronizationContext _context;
+        private static string _appCastURL = @"http://winsparkle.org/example/appcast.xml";
+
+        public WinSparkleDemoForm()
+        {
+            _context = SynchronizationContext.Current;
+            InitializeComponent();
+            
+            var assembly = Assembly.GetExecutingAssembly();
+            var company = (AssemblyCompanyAttribute)assembly.GetCustomAttributes(typeof(AssemblyCompanyAttribute), false)[0];
+            var name = assembly.GetName().Name;
+            var version = assembly.GetName().Version; //Sparkle can parse basic version scheme like X.X.X.X
+
+            WinSparkle.SetAppcastUrl(_appCastURL);
+            WinSparkle.SetAppDetails(company.Company, name, version.ToString());
```

Comment:
> Not very "simple" :/

Seems pretty trivial, actually, judging from the fact that I, who do no .NET development to speak of, was able to find both [how to get the version info](http://stackoverflow.com/questions/10498209/reading-the-version-number-from-a-assemblyinfo-cs-file) and [how to do custom attributes](http://stackoverflow.com/questions/1936953/custom-assembly-attributes) (such as the URL) with a few quick Google searches...